pkg/controller/utils/component.go

```diff
@@ -196,6 +198,24 @@ func (c componentHandler) createOrUpdateObject(ctx context.Context, obj client.O
 				}
 				return nil
 			}
+		case *rbacv1.RoleBinding:
```

Comment:
Probably worth covering this for ClusterRoleBinding as well, but happy for that to be in a second PR